src/internal/confluent/client.go

```diff
@@ -309,6 +309,26 @@ func (c *Client) RegisterSchema(ctx context.Context, clusterId models.ClusterId,
 	return err
 }
 
+func (c *Client) DeleteSchema(ctx context.Context, clusterId models.ClusterId, subject string, schema string, version string) error {
+
+	cluster, err := c.clusters.Get(clusterId)
+
+	if err != nil {
+		return err
+	}
+
+	if len(cluster.SchemaRegistryApiEndpoint) == 0 {
+		return ErrNoSchemaRegistry
+	}
+
+	url := fmt.Sprintf("%s/subjects/%s/versions/%s", cluster.SchemaRegistryApiEndpoint, subject, version)
```

Comment:
I can see that this will delete `latest` version (further down) of the schema for that particular subject. I guess we only ever have one version, as it stands right now.

1. Does the API support deleting the entire subject (including all version)? Do we even what that?
2. What about permanent deletion of subject - is that something we need to consider?